privacy-configuration/tests.json

```diff
@@ -356,6 +371,19 @@
                 "expectFeatureEnabled": true,
                 "exceptPlatforms": [
                     "safari-extension",
+                    "windows-browser",
+                    "android-browser"
```

Comment:
do you have a task for implementing autofill on android?

Reply:
Aye -> https://app.asana.com/0/414730916066338/1201514178819254